rust/operator-binary/src/druid_connection_controller.rs

```diff
@@ -39,36 +41,36 @@ pub enum Error {
     ObjectMissingMetadataForOwnerRef {
         source: stackable_operator::error::Error,
     },
-    #[snafu(display("Failed to get Druid connection string from config map {}", config_map))]
+    #[snafu(display("Failed to get Druid connection string from config map"))]
     GetDruidConnStringConfigMap {
         source: stackable_operator::error::Error,
-        config_map: ObjectRef<ConfigMap>,
```

Comment:
This looks like a secondary object, which we still need to keep around ourselves

Reply:
done